.circleci/config.yml

```diff
@@ -26,7 +25,11 @@ jobs:
           command: hokusai registry push --tag $CIRCLE_SHA1 --force --overwrite
       - run:
           name: Configure
-          command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Validate Kubernetes Yaml
```

Comment:
this is cool – is this something that you are prototyping in this project, and hoping to move to the default templates?

Reply:
+1

Reply:
@dleve123 yes pretty much as a step towards getting config updates integrated with CI